decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -161,14 +140,6 @@ def current_user_unread_data
       end
     end
 
-    def organization_description_label
-      @organization_description_label ||= if empty_organization_description?
-                                            t("decidim.pages.home.footer_sub_hero.footer_sub_hero_body_html")
-                                          else
-                                            decidim_sanitize_admin(translated_attribute(current_organization.description))
-                                          end
-    end
-
     private
 
     def empty_organization_description?
```

Comment:
Heads up, this is used in OrganizationHelper#organization_description_label, so it should be in that file

Reply:
Implemented in https://github.com/decidim/decidim/commit/2109c8bfb3cc3736526b211dc9cc0c26636f47ad